src/main/java/school/lemon/changerequest/java/introduction/hw1/Task5.java

```diff
@@ -3,10 +3,17 @@
 
 public class Task5 {
     public static double calculateA(double x, double y, double z) {
```

Comment:
There are several mistakes:
1. Formula is not correct, please remove extra parentheses, and recheck with task. Using additional variables for part of this huge might help you. 
2. `1 / 2 = 0`. As you may remember, by default all numeric literals are integer. So the result of this operation is integer. There are many ways to solve this issue, you can try you solution in main like this `System.out.println(1/2);` and determine whether it works. I don't want to give you any correct answer, you should figure out it yourself.